compiler/plugins/target/AMD-AIE/aie/AIEDialect.h

```diff
@@ -74,6 +74,8 @@ using DMAChannelDirAttr = mlir::iree_compiler::AMDAIE::DMAChannelDirAttr;
 using AIEArch = mlir::iree_compiler::AMDAIE::AIEArch;
 using AIEDevice = mlir::iree_compiler::AMDAIE::AMDAIEDevice;
 using AIEDeviceAttr = mlir::iree_compiler::AMDAIE::AMDAIEDeviceAttr;
+using AIETargetModel = mlir::iree_compiler::AMDAIE::AMDAIEDeviceModel;
+using TileID = mlir::iree_compiler::AMDAIE::TileLoc;
```

Comment:
Why this change?

Reply:
it simplifies the `sed`s that were/are necessary for AIR which were apparently producing malformed source (I can't find the exact failed CI right now). It's weird because it should have nothing to do with prebuilt LLVM or not but it's a good change nonetheless (because it eliminates/simplifies the `sed`s) so I'm not thinking too hard about it.